src/utils/KeyValues.ts

```diff
@@ -9,4 +10,8 @@ export default class KeyValues {
   public static isNumber (number: string): boolean {
     return this.numbers.test(number)
   }
+
+  public static isEnter (number: string): boolean {
```

Comment:
I guess `number `argument could be a number too. 
`this.enterKeys.includes(String(number))` looks safe

Reply:
@Nikita-Polyakov if we use `key` property it will be string anyway.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key